Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.21.0
```

Comment:
You seem to want to build for the alpine-linux base image. Is there an alpine-linux image with Go support ... I assume `golang:1.21.0-alpine` (that you reference below). That would make more sense, because I am not sure that the flags used in `make` covers *everything* to be static.

Reply:
`golang:1.21.0-alpine` cannot be used because there is no `make` in it.